lib/src/user_license.dart

```diff
@@ -0,0 +1,46 @@
+part of '../lexactivator.dart';
+
+class UserLicense {
+  final int allowedActivations;
```

Comment:
@mumin-khan Now that you have added docs for class members, it should be done across all the classes.

Reply:
I had added it for release file only, since that's deleted, I think we can keep it as it is for now.
:)